siriuspy/siriuspy/sofb/csdev.py

```diff
@@ -16,6 +16,7 @@ class ETypes(_csdev.ETypes):
     """Local enumerate types."""
 
     ENBL_RF = _csdev.ETypes.OFF_ON
+    OPEN_CLOSED = ('OPEN', 'CLOSED')
```

Comment:
Shouldn't it be `OPEN_CLOSED = ('Open', 'Closed')`?

Reply:
putz, verdade!

Reply:
done! Thanks!

Reply:
> putz, verdade!

what do you mean?